### PR TITLE
New software list for the Tandy/Memorex VIS

### PR DESCRIPTION
**Pull request by rzero9**
This adds a new software list for the Tandy/Memorex VIS with 70 entries, which should be almost the entire library of the system. I have tested them but not too extensively, so I may have missed some bugs.

To allow the discs to auto-mount from the command line, this also adds a set_interface line to the Mitsumi ISA CD adapter used by the VIS.

### CODE REVIEW

File: hash/vis.xml [hunk 1]
```diff
@@ -0,0 +1,1265 @@
...
+		</part>
+	</software>
+
+	<software name="kids_paperbag">
+		<!--
+		Origin: redump.org
+		<rom name="Kids Can Read! The Paper Bag Princess - Robert N. Munsch (USA) (En,Es).bin" size="287797776" crc="5d6f9120" sha1="79579e11027766cdc07f6a804d06da075efb34fe"/>
+		<rom name="Kids Can Read! The Paper Bag Princess - Robert N. Munsch (USA) (En,Es).cue" size="136" crc="a25822ea" sha1="519e7a9eec0cd0476f74cf84679b0e5eb0087b62"/>
+		-->
+		<description>Kids Can Read! The Paper Bag Princess - Robert N. Munsch</description>
```
Comment (angelosa) [inline hash/vis.xml:513]: Not sure I like this approach where original book author gets credited inside descriptions. 
It's also inconsistent across the SW itself, cfr. `R. Munsch` for Thomas Swimsuit.
Consider using a separate info XML field or a comment tag.

**Comment by cracyc** [comment]
>Most sections display a "YUV 422" message over a blank screen 

Oooh, interesting...

File: hash/vis.xml (hunk 1 above)
Comment (rzero9) [reply hash/vis.xml]: I have moved the authors' names to a separate book_author info tag. For some reason Github was acting weird and the PR didn't update until I added one more commit after that, though...

**Comment by cracyc** [comment]
https://github.com/mamedev/mame/commit/9ac4ce07cc5f0b9821bb614b797aea60884cb007 should fix all the disc errors caused by audio tracks and the nonworking controls.  It also adds support for the yuv mode used by Columbus but the video mode is still wrong so it's only on a quarter of the screen.  The cdrom changes also seem to have fixed amspace.  Sampler video mode problem is likely still there and survey still hangs, maybe a audio problem since there's choppiness when it plays.

**Pull request merged**